src/test/scala/DockerSuite.scala

```diff
@@ -0,0 +1,143 @@
+import com.typesafe.scalalogging.Logger
+import goatrodeo.omnibor.EdgeType
+import goatrodeo.omnibor.Item
+import goatrodeo.omnibor.ItemMetaData
+import goatrodeo.omnibor.Storage
+import goatrodeo.omnibor.ToProcess
+import goatrodeo.util.FileWrapper
+import org.json4s.JsonAST._
+import org.json4s._
+import org.json4s.native._
+
+import java.io.File
+import scala.collection.immutable.TreeMap
+import scala.collection.immutable.TreeSet
+
+class DockerSuite extends munit.FunSuite {
+  val logger = Logger(getClass())
+
+  test("Can build for a simple Docker file") {
+    val name = "test_data/download/docker_tests/bigtent_2025_03_22_docker.tar"
+
+    val nested = FileWrapper(File(name), name, None)
+    val store1 = ToProcess.buildGraphFromArtifactWrapper(nested)
+
+    val result = store1.purls()
+
+    assertEquals(
+      result,
+      TreeSet("pkg:docker/bigtent@2025_03_22")
```

Comment:
tags can be pretty arbitrary: have you red-teamed what this could be?

Reply:
Just pulled the tags from the `manifest.json` file and put it into the pURL. If there's some checking that should be done, I'm down for that... but right now, just copying the data over.

Reply:
Aha, [found where this is defined](https://github.com/distribution/reference/blob/main/reference.go#L18): `/[\w][\w.-]{0,127}/` matches tags.